src/analytics/follow_notification_link.rs

```diff
@@ -0,0 +1,54 @@
+use {
+    crate::model::types::AccountId,
+    parquet_derive::ParquetRecordWriter,
+    relay_rpc::domain::{ProjectId, Topic},
+    serde::Serialize,
+    std::sync::Arc,
+    uuid::Uuid,
+};
+
+pub struct FollowNotificationLinkParams {
+    pub project_pk: Uuid,
```

Comment:
this looks good, would add geo-data and user-agent data ( e.g OS / browser ) stuff. 

We'd probably like to see in the future CTR by different cohorts and whatnot.. for mvp this is awesome

Reply:
Added!